reading_order_widget.py

```
@@ -50,159 +63,348 @@ def __init__(self):
+            comic_id (str): The ID of the comic to check.
 
-    def contains_comic(self, comic_id: str) -> bool:
+        Returns:
+            bool: True if already in the list, False if not.
+        """
         for i in range(self.count()):
             if self.item(i).data(Qt.ItemDataRole.UserRole) == comic_id:
                 return True
         return False
 
+    def startDrag(self, supportedActions) -> None:
```

Ruff (0.15.12)

[warning] Unused method argument: `supportedActions`

(ARG002)